src/modal/modal-window.ts

```diff
@@ -15,7 +15,7 @@ import {ModalDismissReasons} from './modal-dismiss-reasons';
 @Component({
   selector: 'ngb-modal-window',
   host: {
-    'class': 'modal fade in',
+    '[class]': '"modal fade in" + (windowClass ? " " + windowClass : "")',
```

Comment:
This is part of the host selectors applied to the angular template. AFAIK there's no way to use ES6 string concatenation here. It follows the same pattern that's used on line 26: `<div [class]="'modal-dialog' + (size ? ' modal-' + size : '')" role="document">`